### PR TITLE
Log send include (keybase) system log

### PR DESCRIPTION
**Pull request by gabriel**
We have a log file (on macOS) for the privileged tasks (like fuse install). This includes it in log send.

Also:
- Move some platform specific log stuff into install since it's install specific
- Don't show warning on log file not found, if the web don't have the log, then it wasn't there

### CODE REVIEW

**Comment by gabriel** [comment]
@zanderz @maxtaco

**Comment by maxtaco** [comment]
Looks like the windows build failed ^

File: go/install/install_windows.go [hunk 1]
```diff
@@ -41,3 +41,151 @@ func RunApp(context Context, log Log) error {
...
 	return nil
 }
+
+
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build windows
+
+package client
```
Comment (maxtaco) [inline go/install/install_windows.go]: this likely has to change to `install`

File: go/install/install_windows.go [hunk 2]
```diff
@@ -41,3 +41,151 @@ func RunApp(context Context, log Log) error {
...
+		if strings.Contains(scanner.Text(), "File System") {
+			logFile.WriteString(scanner.Text() + "\n")
+		}
+	}
+	logFile.WriteString("\n\n")
+}
+
+func SystemLogPath() string {
+	return ""
+}
```
Comment (maxtaco) [inline go/install/install_windows.go]: Definitely needs to be tested on windows before it's merged..

File: go/install/install_windows.go (hunk 1 above)
Comment (gabriel) [inline go/install/install_windows.go]: oops, yeah didn't merge the 2 files correctly

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 667ef143dc65...9a586511d390 introduced some visual changes on linux:
 * Changed: **New_Proof_Post-Facebook** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474479061372-667ef143dc65-9a586511d390-linux/New_Proof_Post-Facebook-667ef143dc65.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474479061372-667ef143dc65-9a586511d390-linux/New_Proof_Post-Facebook-9a586511d390.png)  
   New_Proof_Post-Facebook comparison

**Comment by gabriel** [comment]
Tested on windows and is all good.

**Pull request merged**